### DISCUSSION TITLE
kuka kr 240 error

### DISCUSSION THREAD

**Discussion by francescobic** [visose/Robots#94] [Q&A] [answered]
Hi
i have prepared a definition to control a kuka kr 240, from grasshopper i don't have any error but when i copy files in program folder on kuka, it show me error in many lines (line 3, 4, 6, 7 and 9).

**Comment by visose** [comment, marked as answer]
Can you show any of those lines? What is the error?
Make sure you don't use spaces or strange characters in variable names, like the tool or program name. Older KUKAs only allow for short names, see this issue: #20

**Comment by visose** [comment]
I assume that fixed the issue, feel free to open it again if not with the error messages you got.
